fs/copy.go

```diff
@@ -63,7 +63,7 @@ func copyFile(source, destination string) error {
 		return err
 	}
 
-	err = destinationFile.Chmod(info.Mode())
+	err = os.Chmod(destination, info.Mode())
```

Comment:
`File.Chmod` causes a syscall error for Windows. See related issue: https://github.com/google/skicka/issues/77#issuecomment-97988722